src/python/bot/fuzzers/libFuzzer/fuzzer.py

```diff
@@ -60,3 +61,6 @@ def generate_arguments(self, fuzzer_path):
             '%s%d' % (constants.RSS_LIMIT_FLAG, constants.DEFAULT_RSS_LIMIT_MB))
 
     return ' '.join(arguments)
+
+  def get_mutator_plugin(self, fuzzer_binary_name):
+    mutator_plugin.get_mutator_plugin(fuzzer_binary_name)
```

Comment:
we should move away from the model we have of methods setting some global state (in this case a env var). we should return the path to this plugin instead.

this will also make it easier if we decide to support this as an argument in libFuzzer itself.

Reply:
Done.